templates/01_block-scoping.test.js

```diff
@@ -15,16 +15,19 @@ test('can be used in place of `var`', () => {
   expect(isBestBand).toBe(true)
 })
 
-test('can modify the value of a `let` variable', () => {
+test('can modify the value of a `let` variable even in the next block statement', () => {
   // WORKSHOP_START
   // Declare 'releaseName' using 'let', setting the value to 'ES6'
-  // Change value of releaseName to be `ES2015`, the new name for ES6
+  // Change value of releaseName to be `ES2015`, the new name for ES6 wrapped in a block statement
+  expect(releaseName).toBe(/* ENTER YOUR GUESS HERE */)
   // WORKSHOP_END
   // FINAL_START
   let releaseName = 'ES6'
-  releaseName = 'ES2015'
-  // FINAL_END
+  {
+    releaseName = 'ES2015'
+  }
   expect(releaseName).toBe('ES2015')
+  // FINAL_END
```

Comment:
Hmmm... I'm thinking this test might be a little confusing now that I think about it. Perhaps we could do something like this instead:

```javascript
test('can modify the value of a `let` variable even in the next block statement', () => {
  let releaseName = 'ES6'
  {
    releaseName = 'ES2015'
  }
  // WORKSHOP_START
  expect(releaseName).toBe(/* ENTER YOUR GUESS HERE */)
  // WORKSHOP_END
  // FINAL_START
  expect(releaseName).toBe('ES2015')
  // FINAL_END
})
```

This way it's a little less ambiguous of what they're learning. Thoughts?